ember-cli-build.js

```diff
@@ -16,6 +16,15 @@ module.exports = function (defaults) {
     prember: {
       urls: ['/', '/about-me', '/contact', '/posts', ...posts],
     },
+    'responsive-image': {
```

Comment:
you need to add `responsive: false` to this config a la https://github.com/simonihmig/ember-responsive-image#configuration